README.md

```diff
@@ -47,13 +47,13 @@ yarn build
 Start a **storybook** server for the polaris-react components
 
 ```sh
-yarn workspace @shopify/polaris dev
+yarn turbo run dev --filter=@shopify/polaris...
```

Comment:
`...` TIL https://turborepo.org/docs/core-concepts/filtering#include-dependents-of-matched-packages